controllers/controllers.errors.js

```diff
@@ -7,7 +7,21 @@ const handlerError500 = (err, req, res, next) => {
     res.status(500).send({msg: "server error"})
 }
 
+const handlerError400 = (err, req, res, next) => {
```

Comment:
I think this function name could be a little clearer eg `handle400`

Reply:
done